Hero/Hero.swift

```diff
@@ -86,6 +86,20 @@ internal extension Hero {
   }
 }
 
+fileprivate extension Hero {
+  func closureProcessForHeroDelegate<T: UIViewController>(vc: T, closure: (HeroViewControllerDelegate)->()) {
+    switch vc {
+      case let fvc as HeroViewControllerDelegate:
+        closure(fvc)
```

Comment:
@lkzhao If you add "fallthough", then the existing "if" logic would be more readable. I will change that part to the existing logic.